src/global/global.c

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// This library
+#include <k4ainternal/global.h>
+
+// System dependencies
+#ifdef _WIN32
+#include <windows.h>
+#else
+#include <pthread.h>
+#endif
+#include <assert.h>
+
+#ifdef _WIN32
+
+C_ASSERT(sizeof(k4a_init_once_t) == sizeof(INIT_ONCE));
+
+static BOOL CALLBACK InitGlobalFunction(PINIT_ONCE InitOnce, PVOID Parameter, PVOID *lpContext)
```

Comment:
nit: The rest of the library uses `bool` not `BOOL`. 

```suggestion
static bool CALLBACK InitGlobalFunction(PINIT_ONCE InitOnce, PVOID Parameter, PVOID *lpContext)
```

Reply:
It's a windows.h defined function prototype, so matching their standard seems correct